imagepipeline-base/src/main/java/com/facebook/cache/disk/DynamicDefaultDiskStorage.java

```diff
@@ -71,6 +71,15 @@ public boolean isEnabled() {
       return false;
     }
   }
+ 
+ @Override
+  public boolean isExternal() {
+    try {
+      return get().isExternal();
```

Comment:
Hello,
@plamenko, here I think it's get() that throws IOException : 
` @VisibleForTesting
  /* package protected */ synchronized DiskStorage get() throws IOException {
...
}`
?